test/integration/patients/patient/archive.js

```diff
@@ -463,7 +463,9 @@ context('patient archive page', function() {
       })
       .visitOnClock(`/patient/archive/${ testPatient.id }`, { now: testTs() })
       .wait('@routePatient')
-      .wait('@routePatientFlows');
+      .wait('@routePatientActions')
+      .wait('@routePatientFlows')
+      .wait(200); // for initial ws connection to be created
```

Comment:
Without this `cy.wait()`, these tests fail ~50% of the time. I think what's happening is the `mockWs()` command isn't always finished before we send the first ws message in the test via `cy.sendWs()`.

Adding a `cy.wait(200)` seems to fix the issue, where the test passes 100% of the time. I don't think this does much harm. 

But I would prefer to not do this if there was another way to fix it. The `mockWs` would have to be reconfigured somehow, to somehow let us know when the initial socket connection is ready.

Reply:
```js
    cy
      .get('@wsHandleMessage')
      .should('have.been.calledOnce');
```
will do it.

The issue was that the `send` is happening in Cypress before the `subscribe` happens from the app.. so we have to wait until the subscribe is called.